### ISSUE TITLE
TreeTable: Resizing one column resizes other columns as well

### ISSUE THREAD

**Issue by B34v0n** [primefaces/primeng#15876]
### Describe the bug

When trying to resize one specific column in the TreeTable, it resizes the others as well.

You can see the issue in https://primeng.org/treetable#scrollable, I created a gif:

![columnMoves](https://github.com/primefaces/primeng/assets/88542001/4b6c365f-a63b-4b27-82b3-d67eb6154bf2)

Everytime I make the `name`-column bigger or smaller, the `type`-column moves a bit to the right.

In my local project, the issue is even bigger, because all of my remaining (14) columns get bigger when I resize one.

Also: shouldn't the `size`-column stay the same size as it is, when I resize the `name`-colum? 

### Environment

Chrome

### Angular version

17.3.10

### PrimeNG version

17.18.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v20.13.1

### Browser(s)

Chrome Version 126.0.6478.114

### Steps to reproduce the behavior

1. Go To  https://primeng.org/treetable#scrollable
2. Resize `name`-column
3. `type`-column moves a little bit to the right, `size`-column gets smaller/bigger

### Expected behavior

size-column should stay the size that it was resized to.
type-column should not change it's position ever so slightly.

**Comment by B34v0n** [comment]
I also found the solution for this issue.

In the treetable.ts-File there are five rows, which are too much.
https://github.com/primefaces/primeng/blob/ceedbb9faceba0fd49e1cf220216c1a2c10d6e7d/src/app/components/treetable/treetable.ts#L1442-L1446

The resizing is already done within the method "setWidth" so it doesn't need to be done before that.
Doing it twice resulted in adding the delta twice, which resulted in changing all column-widths.

If I remove these lines, the treetable works as expected.

I tried to push a branch, but wasn't able to. Can someone push that solution, please?
